### PR TITLE
Add the second part of the ?event query to toriel

### PR DESCRIPTION
**Pull request by tregsthedev**

### CODE REVIEW

File: util/transcript.yml [hunk 1]
```diff
@@ -91,7 +97,7 @@ block:
           text: ${this[1].text}
         value: ${this[1].value}
 
-announcements-to-cave: hello wanderer! head to <#${this.t('channels.cave')}> to continue your journey.
+announcements-to-cave: hello wanderer! head to <#${this.t('channels.arav-tor')}> to continue your journey.
```
Comment (sampoder) [inline util/transcript.yml:100]:
```suggestion
announcements-to-cave: hello wanderer! head to <#${this.t('channels.cave')}> to continue your journey.
```

File: util/transcript.yml [hunk 2]
```diff
@@ -58,6 +58,12 @@ channels:
   wordle: C02TWKX227J
   toriel-dev: C02B7CWDD0E
   angelhacks: C050R91GSF5
+  arv-tor: C054EE209T7
+  angelhacks-toronto: C051XV7CJLW
+  angelhacks-ba: C051MQSF970
+  angelhacks-la: C051DU6T71V
+  
+
```
Comment (sampoder) [inline util/transcript.yml:66]:
```suggestion
```

File: index.js [hunk 3]
```diff
@@ -257,12 +252,13 @@ app.action(/.*?/, async (args) => {
...
       })
-      if (invite?.welcome_message == "I'm going to Epoch!") {
+      if (invite?.event) {
+        const ref = invite?.event
         await prisma.user.update({
           where: { user_id: user },
           data: { club_leader: false },
         })
-        await addToChannels(user, true)
+        await addToChannels(user, true, ref)
```
Comment (sampoder) [inline index.js:261]: We should simplify this and pass one string param like:

await addToChannels(user, event) // event = invite?.event

The code above should still work as long as ref is replaced with event

File: util/transcript.yml [hunk 4]
```diff
@@ -58,6 +58,12 @@ channels:
   wordle: C02TWKX227J
   toriel-dev: C02B7CWDD0E
   angelhacks: C050R91GSF5
+  arv-tor: C054EE209T7
```
Comment (sampoder) [inline util/transcript.yml:61]:
```suggestion
```

File: index.js (hunk 3 above)
Comment (tregsthedev) [reply index.js]: That makes more sense; I'm not sure why I made another param

**Review by sampoder — APPROVED** [review]
LGTM!

**Comment by tregsthedev** [comment]
I forgot to add one thing, sumbitting a pr later for it

**Comment by tregsthedev** [comment]
it's not related to this PR though, everything for this is set

**Pull request merged**